### ISSUE TITLE
Completion and workover VF double counted on GHG summary sheet

### ISSUE THREAD

**Issue by qlangfitt** [arbrandt/OPGEE#278]
On the GHG summary sheet, the VF emissions from completions and workovers are listed under Drilling and Development (venting column). However, the on the Venting and Fugitives 3.0 sheet, they are classified as Production & Extraction. Thus, they are also included in the Production & Extraction row on the GHG summary sheet (fugitives column).

Proper classification should be done for completions and workovers so that they only appear once on the GHG summary sheet.

Note that although the emissions are double counted on GHG Summary, since drilling and development VFF emissions are forced at 0 on the Active Field sheet, the double counting does not carry into the final CI.

**Comment by arbrandt** [comment]
I updated the GHG summary sheet in response to issue #279. Now there is no "NEW" GHG summary sheet and the only one now has the proper process level detail to line up with the energy summary sheet. Please check if the issue persists in the new sheet.

**Comment by qlangfitt** [comment]
The issue of double counting on the GHG summary sheet is solved, however, they are still listed incorrectly on the Venting & Fugitives 3.0 sheet. Since they are listed under "Drilling & Development" as vented emissions for the GHG summary sheet, I think they should be reclassified on the Venting & Fugitives 3.0 sheet to be consistent. Currently, they are assigned as fugitive emissions under "Production & Extraction" on that sheet.

**Comment by qlangfitt** [comment]
Actually, I have to take back my last statement that it was solved. I just looked closer and they are still double counted. VF emissions from workovers and completions are included on both the Drilling & Development sheet and the Well & Downhole pump sheet. I like the way it was done on the Well & Downhole pump sheet better, so suggest keeping the calculations as are, except zeroing out the VF portion on Drilling & Development (maybe add a note underneath it that workovers and completions are handled on a different sheet).
